MoreLinq.Test/InterleaveTest.cs

```diff
@@ -35,6 +36,41 @@ public void TestInterleaveIsLazy()
             new BreakingSequence<int>().Interleave(new BreakingSequence<int>());
         }
 
+        /// <summary>
+        /// Verify that Interleave early throw ArgumentNullException when an element
+        /// of otherSequences is null.
+        /// </summary>
+        [Test]
+        public void TestInterleaveEarlyThrowOnNullElementInOtherSequences()
```

Comment:
This test seems unrelated to this PR so best to move it to another.

Reply:
Fixed in 7d0aa0d and a7f2c6f